### PR TITLE
Add more checks in the workloads scripts

### PR DESCRIPTION
**Pull request by amitsagtani97**
# Description
Adding more checks for confirmation of pod creation and running in workload scripts for better error debugging.

### CODE REVIEW

**Review by mohit-sheth — APPROVED** [review]
LGTM

File: workloads/scale-perf/run_scale_fromgit.sh [hunk 1]
```diff
@@ -55,19 +56,38 @@ spec:
...
+      if [ $? -ne 0 ]; 
+      then 
+        exit 1
+      fi
+
+      uuid=${long_uuid:0:8}
 
-      sleep 30
+      # Checks the presence of scale pod. Should exit if pod is not available.
+      scale_pod=$(get_pod "app=scale-$uuid" 300)
```
Comment (chaitanyaenr) [inline workloads/scale-perf/run_scale_fromgit.sh:69]: Maybe also check if the pod is in running state?

File: utils/common.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  done
+  echo $uuid
+  return 0
+}
+
+# The argument is 'timeout and 'pod name' in seconds'
+function check_pod_ready_state () {
+  pod_name=$1
+  timeout=$2
+  kubectl wait --for=condition=ready pods --namespace ${3:-my-ripsaw} $pod_name --timeout=$timeout
```
Comment (chaitanyaenr) [inline utils/common.sh:52]: Maybe use oc instead of kubectl to be consistent? Thoughts?

File: utils/common.sh [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      echo "Unable to fetch the benchmark uuid!"
+      return 1
+    fi
+  done
+  echo $uuid
+  return 0
+}
+
+# The argument is 'timeout and 'pod name' in seconds'
+function check_pod_ready_state () {
```
Comment (chaitanyaenr) [inline utils/common.sh:49]: Maybe log that the script is waiting for x seconds for the pod to transition to ready state? Thoughts?

**Review by chaitanyaenr — APPROVED** [review]
LGTM, nice work @amitsagtani97.

**Pull request merged**